charts/osm-arc/templates/metrics-rbac.yaml

```diff
@@ -53,3 +53,36 @@ subjects:
   - kind: ServiceAccount
     name: {{ .Release.Name }}
     namespace: {{ .Release.Namespace }}
+---
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: metrics-agent-cluster-role
+rules:
+- apiGroups: [""]
+  resources: ["namespaces"]
+  verbs: ["get", "patch"]
+{{- if eq .Values.Azure.Cluster.Distribution "openshift" }}
+- apiGroups:
+  - security.openshift.io
+  resourceNames:
+    - privileged
+  resources:
+    - securitycontextconstraints
+  verbs:
+    - use
```

Comment:
is there a reason this formatting is different than the usual rbac formatting? ex: verbs: ["use"]

Reply:
I was following the formatting we use in osm-label for OpenShift SCC, as well as what was outlined on https://docs.openshift.com/container-platform/4.6/authentication/managing-security-context-constraints.html. Maybe we can reformat both of these in a separate PR.

Reply:
hm I believe that's usually just how rbac prints out when you view it using kubectl and that's what they're showing as role definition in the example you linked as well. I think it would make sense to keep the formatting consistent

Reply:
however since we use it in osm-rbac as well I guess it can also just stay consistent with that